lib/transformations/index.js

```diff
@@ -38,7 +38,8 @@ function transform(source, transforms, options) {
 	}, options);
 	transforms = transforms || Object.keys(transformations).map(k => transformations[k]);
 	transforms.forEach(f => f(jscodeshift, ast));
-	return ast.toSource(recastOptions);
+	const astStr = ast.toSource(recastOptions);
+	return prettier.format(astStr, {singleQuote: true, tabWidth: 2});
```

Comment:
Lets drop recast options. Recast pretty print is not reliable. I'll add prettier options instead